app/controllers/expense_categories_controller.rb

```diff
@@ -6,6 +6,7 @@ def new
 
   def create
     @expense_category = @user.expense_categories.new(expense_category_params)
+    @expense_category.user = @user
     if @expense_category.save
       flash[:success] = ' Category created '
```

Comment:
- [ ] Hello, I would advise you to aslo create success and fail paths update and destroy actions.